src/etc/inc/services.inc

```diff
@@ -158,7 +158,7 @@ function services_radvd_configure($blacklist = array()) {
 				$radvdconf .= "\tAdvOtherConfigFlag on;\n";
 				break;
 		}
-		$radvdconf .= "\tprefix {$subnetv6}/{$ifcfgsnv6} {\n";
+		$radvdconf .= "\tprefix {$ifcfgipv6}/{$ifcfgsnv6} {\n";
```

Comment:
Why use the interface address here? As mentioned in the main comments, that doesn't seem to be correct

Reply:
Fixed
Now it uses full ip address only for unmanaged mode